src/Tizen.NUI/src/internal/Xaml/CreateValuesVisitor.cs

```diff
@@ -448,21 +470,15 @@ static object CreateLanguagePrimitive(Type nodeType, IElementNode node)
                 }
                 if (nodeType == typeof(Int16))
                 {
-                    short retval;
-                    if (short.TryParse(valuestring, NumberStyles.Number, CultureInfo.InvariantCulture, out retval))
-                        return retval;
+                    return (Int16)ConvertScriptToPixel(valuestring);
```

Comment:
How about call GraphicsTypeManager.Instance.ConvertScriptToPixel(valuestring) directly ?

Reply:
I was thought that way firstly.. 
but double and Int64, Uint64 may overflow.(even though that value never overflowed in our gui systems mostly)

Reply:
How about change to return double type?

`double GraphicsTypeManager.Instance.ConvertScriptToPixel(string valuestring)`

Reply:
I change GraphicsTypeManager.Instance.ConvertScriptToPixel(valuestring) all.

Thinking of overflow possibility,
we are getting value from string by user which is very hard to put overflowed value,
also our graphics system are based on float, so it will overflow somewhere else at the end
even we try to prevent well in parse.

so I think it is good enough to ignore overflow possiblity in this case.